exercises/grade-school/src/test/java/SchoolTest.java

```diff
@@ -14,119 +14,89 @@
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertArrayEquals;
 
+/*
+ * Based on problem-specifications/exercises/grade-school/canonical-data.json 1.0.0
+ */
```

Comment:
Thats fair. I included it because I had seen it in a Python or c# test, I believe, but I don't think that one had the .meta directory with the version file.

Reply:
I see! Yes, we used to use them as well before we had the version files 🙂